core/chains/evm/config/toml/defaults/Shibarium_Mainnet.toml

```diff
@@ -0,0 +1,38 @@
+ChainID = '109'
+FinalityTagEnabled = true
+LinkContractAddress = '0x71052BAe71C25C78E37fD12E5ff1101A71d9018F'
+LogPollInterval = '1s'
+MinIncomingConfirmations = 5
+NoNewHeadsThreshold = '30s'
+RPCBlockQueryDelay = 10
+RPCDefaultBatchSize = 100
+
+[Transactions]
+ResendAfterThreshold = '1m'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+EIP1559DynamicFees = true
+PriceDefault = '10 gwei'
+PriceMin = '2 gwei'
```

Comment:
Since EIP1559 is enabled, I don't think `PriceDefault` and `PriceMin` are controlling anything. The equivalent for dynamic transactions is `TipCapDefault` and `TipCapMin` if we want to set these limits.

Reply:
Fixed. ty